share/spice/movies/cast/movies_cast.js

```diff
@@ -0,0 +1,75 @@
+(function (env) {
+    'use strict';
+
+    // This is needed to prevent jQuery from adding '_={timestamp}' to the end
+    // of the URL, which breaks the call to getScript()
+    $.ajaxSetup({ cache: true });
+
+    var DETAIL_ENDPOINT = '/js/spice/movies/details/';
+    var POSTER_URL = 'http://image.tmdb.org/t/p/w185';
+
+    var getMovieUrl = function(id) {
+        return 'http://www.themoviedb.org/movie/' + id;
+    }
+    Handlebars.registerHelper("getMovieUrl", getMovieUrl);
+
+    env.ddg_spice_movies_cast = function(data) {
+        // This isn't exactly pretty, but I can't think of any
+        // more graceful way to handle this.
+        var hasFailed = (
+            !data ||
+            !data.results ||
+            !data.results[0] ||
+            !data.results[0].id
+        );
+
+        if (hasFailed) {
+            return Spice.failed('cast');
+        }
+
+        $.getScript(DETAIL_ENDPOINT + data.results[0].id)
```

Comment:
Technically this is fine, but personally I really dislike this approach (I know we have examples of this in our existing code...) because it's harder to follow the logic IMO.

I would suggest we stick with one callback function and use:

```js
$.getJSON(DETAIL_ENDPOINT + data.results[0].id, function(){
  //error checking
  Spice.add({...});
});
```